packages/ses/src/permits.js

```diff
@@ -270,6 +270,7 @@ export const FunctionInstance = {
   // Do not specify "prototype" here, since only Function instances that can
   // be used as a constructor have a prototype property. For constructors,
   // since prototype properties are instance-specific, we define it there.
+  // The exception to this is Hermes, fixed since in Static Hermes.
```

Comment:
I don't know what "Static Hermes" is.  add reference to comment?

Reply:
added here https://github.com/endojs/endo/pull/2563/commits/1d4bd35424d312e864109378f0ee0c8bab4f9ced

Reply:
> @leotm Are we testing against Hermes in CI somewhere? Can you point me to it?

both proposed 2 changes (permits.js, intrinsics.js) can be verified working against Hermes CI in
https://github.com/endojs/endo/tree/ses-hermes-whitelistIntrinsics-repro
running `yarn build:hermes && yarn test:hermes` where completePrototypes is now fixed on Hermes

otherwise final changes to https://github.com/endojs/endo/pull/2334 nearly ready for official Hermes CI feedback